addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/ClientScriptBasedAuthenticationMethodType.java

```diff
@@ -355,8 +361,14 @@ public WebSession authenticate(
                 }
 
                 if (authScript instanceof ZestAuthenticationRunner zestRunner) {
+                    ZestScript zestScript = zestRunner.getScript().getZestScript();
+                    if (!hasBrowserLaunch(zestScript)) {
+                        LOGGER.warn("The script does not have the expected browser launch.");
```

Comment:
Instead of "expected", how about "any" or "any top level".

In my opinion "expected" implies looking for particular properties or something....so there might be one but not the "expected" one..... sorry if I'm not being clear.

Reply:
Agreed it might be read that way, will change to any.